great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -321,61 +325,52 @@ def _get_bootstrap_confidence_interval(
-            "BootstrapResult", ["confidence_interval", "standard_error"]
-        )
-
-        bootstrap_result: BootstrapResult
+        n_resamples: int
         if num_bootstrap_samples is None:
-            bootstrap_result = bootstrap(
-                bootstrap_samples,
-                np.mean,
-                vectorized=False,
-                confidence_level=confidence_level,
-                random_state=rng,
-                method="basic",
-            )
+            n_resamples = DEFAULT_BOOTSTRAP_NUM_RESAMPLES
         else:
-            bootstrap_result = bootstrap(
-                bootstrap_samples,
-                np.mean,
-                vectorized=False,
-                confidence_level=confidence_level,
-                n_resamples=num_bootstrap_samples,
-                random_state=rng,
-                method="basic",
-            )
+            n_resamples = num_bootstrap_samples
 
-        confidence_interval: ConfidenceInterval = bootstrap_result.confidence_interval
-
-        confidence_interval_low: np.float64 = confidence_interval.low
-        confidence_interval_high: np.float64 = confidence_interval.high
-
-        std: Union[np.ndarray, np.float64] = bootstrap_result.standard_error
-
-        stds_multiplier: np.float64 = NP_SQRT_2 * special.erfinv(confidence_level)
-        margin_of_error: np.float64 = stds_multiplier * std
+        return self._compute_bootstrap_estimation_and_return_confidence_interval(
+            metric_values=metric_values,
+            confidence_level=confidence_level,
+            n_resamples=n_resamples,
+        )
 
-        confidence_interval_low -= margin_of_error
-        confidence_interval_high += margin_of_error
+    @staticmethod
+    def _compute_bootstrap_estimation_and_return_confidence_interval(
+        metric_values: np.ndarray,
+        confidence_level: np.float64,
+        n_resamples: int,
+    ):
+        uncertainty_level: float = 1.0 - confidence_level
+        quantile_probability_lower: float = 5.0e-1 * uncertainty_level
+        quantile_probability_upper: float = 1.0 - quantile_probability_lower
 
-        return ConfidenceInterval(
-            low=confidence_interval_low, high=confidence_interval_high
+        bootstraps: np.ndarray = np.random.choice(
+            metric_values, size=(n_resamples, metric_values.size)
+        )
+        low: np.float64 = np.mean(
+            np.quantile(
+                bootstraps,
+                axis=1,
+                q=quantile_probability_lower,
+                interpolation="linear",  # can be omitted ("linear" is default)
+            )
+        )
+        high: np.float64 = np.mean(
+            np.quantile(
+                bootstraps,
+                axis=1,
+                q=quantile_probability_upper,
+                interpolation="linear",  # can be omitted ("linear" is default)
+            )
         )
+        return ConfidenceInterval(low=low, high=high)
```

Comment:
This is no longer a confidence interval. Please return simply (low, high) (and remove all other references to confidence_interval per notes above.

Reply:
Done!